components/expenses/PayoutMethodForm.js

```diff
@@ -114,7 +114,7 @@ const PayoutMethodForm = ({ payoutMethod, fieldsPrefix, host }) => {
       {payoutMethod.type === PayoutMethodType.BANK_ACCOUNT && host && (
         <PayoutBankInformationForm isNew={isNew} getFieldName={getFieldName} host={host} />
       )}
-      {isNew && (
+      {isNew && host?.transferwise && (
```

Comment:
I think this is wrong, even if `host?.transferwise` is null, we're fetching the available currencies using the Platform credentials and normally able to display the form.

Reply:
Makes sense. I added this initially to not show the user the small save checkbox when Wise is not setup. But then you are right and this isn't needed when we are able to fetch the available currencies. One thing though is that in dev when Wise is not configured we will see the warning and the checkbox; which I think is fine for now.  🤔 

![image](https://user-images.githubusercontent.com/12435965/133269591-4a15a48f-f1e2-44d5-bc79-1fda836b5e29.png)